validator/delphi_validator/report.py

```diff
@@ -0,0 +1,99 @@
+"""Validation output reports."""
+import sys
+from datetime import date, datetime
+from typing import List, Tuple
+
+class ValidationReport:
+    """Class for reporting the results of validation."""
+    def __init__(self, errors_to_suppress: List[Tuple[str]]):
+        """Initialize a ValidationReport.
+        Parameters
+        ----------
+        errors_to_suppress: List[Tuple[str]]
+            List of error identifications to ignore.
+
+        Attributes
+        ----------
+        errors_to_suppress: List[Tuple[str]]
+            See above
+        num_suppressed: int
+            Number of errors suppressed
+        total_checks: int
+            Number of validation checks performed
+        raised_errors: List[Exception]
+            Errors raised from validation failures
+        raised_warnings: List[Exception]
+            Warnings raised from validation execution
+        suppressed_errors: List[Exception]
+            Errors raised from validation failures not found in `self.errors_to_suppress`
```

Comment:
Should this be unsuppressed errors? Since it's errors that are not in the ignore list.

Reply:
Yes, done.